flatliners/__init__.py

```diff
@@ -9,3 +9,4 @@
 from .weirdnessscore import WeirdnessScore
 from .influxdbstorage import InfluxdbStorage
 from .singlevaluemetric import SingleValueMetric
+from .prometheusendpoint import PrometheusEndpoint
```

Comment:
stickler is aways complaining.

could you check if we can remove all the imports here?
if so, remove them in another PR...

Reply:
@fridex should we just disable F401 for this kind of linting message? Or you think something with the structure of our code is off?